mautic/mautic
# Bugfix Segment filter "email read count equals 0"  and new source ID filter

Fix email read count with 0

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5041
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Bugfix for https://github.com/mautic/mautic/issues/5041 

#### Steps to test this PR:
1.  Create filter with email read count equals = 0
2.  Run `php app/console m:s:r`
3. Works if segment is not empty
4. Create filter with Page source and Page source ID (for example email and id of email )
5.  Run `php app/console m:s:r`
6. See if segments contains contacts with contacts hits pages from that email
